internal/exec/stack_processor_utils.go

```
@@ -25,8 +25,16 @@
 
 	// Mutex to serialize updates of the result map of ProcessYAMLConfigFiles function
 	processYAMLConfigFilesLock = &sync.Mutex{}
+
+	// Add at the top of the file with other global variables
+	globalStacksBasePath string
```

golangci-lint (1.62.2)

var `globalStacksBasePath` is unused

(unused)